backend/core/test/applications/applications.e2e-spec.ts

```diff
@@ -589,21 +591,6 @@ describe("Applications", () => {
       .expect(400)
   })
 
-  it(`should disallow a user to send too much application submits`, async () => {
```

Comment:
we could ultimately from discussion this test is really testing a piece of nestjs not our code. Specifically this tests the throttler which we only pass some arguments into instead of implementing ourselves.

As long as nestjs's throttle was working so would this test, and if something went wrong with it, its an issue with nestjs's throttle instead of our code